src/puppetlabs/puppetdb/command.clj

```diff
@@ -467,7 +470,9 @@
                "deactivate node" exec-deactivate-node
                "configure expiration" exec-configure-expiration
                "replace catalog inputs" exec-replace-catalog-inputs)]
-    (exec cmd start db conn-status)))
+    (if (= command "store report")
```

Comment:
It looks like `exec-store-report` is only called in the `exec-command` func. If we make the config value the first argument to `exec-store-report` we could then use `(partial exec-store-report blocklist-config)` in the `case` statement above and wouldn't need to add the if condition for `"store report"` at the bottom of this func.